### PR TITLE
[FEC-54]-date-pickers

### PR DESCRIPTION
**Pull request by YuriyPanasiuk**
> I've decided just add the most popular libraries and add links for documentation without adding code examples

1.  [Well-known date and time picker libraries](#well-known-date-and-time-picker-libraries)
    - [React Datepicker](#react-datepicker)
    - [MUI Pickers](#mui-pickers)
    - [React-Dates](#react-dates)
    - [React-DateTime-Picker](#react-datetime-picker)
    - [React-Day-Picker](#react-day-picker)
2.  [Date utility libraries](#date-utility-libraries)
    - [Date-fns](#date-fns)
    - [Luxon](#luxon)
    - [Day.js](#day.js)
3.  [Summary](#summary)

### CODE REVIEW

File: ui-components/date-time-pickers.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+- Support for disabling specific dates or date ranges.
+- Easy integration with React forms.
+
+Documentation: https://reactdatepicker.com/
+
+---
+
+### MUI Pickers
+
+**MUI Pickers** is an extension of the MUI library that provides reusable and customizable date and time pickers with a Material Design look and feel.
```
Comment (yarikhot) [inline ui-components/date-time-pickers.md:37]: Please note that `DateRangePicker` is paid feature for the MUI picker

File: ui-components/date-time-pickers.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+- The API might be slightly different from Moment.js, which could require some adjustments if migrating from Moment.js.
+- Less popular compared to Moment.js (although popularity has been growing steadily).
+
+GitHub Repository: https://github.com/date-fns/date-fns
+Official Website: https://date-fns.org/
+
+---
+
+### Moment.js
```
Comment (yarikhot) [inline ui-components/date-time-pickers.md:122]: as I know the moment is deprecated, not sure we need use one in the new projects
WDYT?
I`d like to remove this section or mark it as **deprecated**

Comment (YuriyPanasiuk) [reply ui-components/date-time-pickers.md:122]: Yeah, I didn't know about that). I'll change to [Luxon](https://moment.github.io/luxon/#/?id=luxon)

**Review by yarikhot — APPROVED** [review]
🔥

**Review by quatrolavita — APPROVED** [review]

**Pull request merged**